hudi-integ-test/src/test/java/org/apache/hudi/integ/ITTestBase.java

```diff
@@ -236,8 +238,11 @@ private TestExecStartResultCallback executeCommandInDocker(String containerName,
     int exitCode = dockerClient.inspectExecCmd(createCmdResponse.getId()).exec().getExitCode();
     LOG.info("Exit code for command : " + exitCode);
     if (exitCode != 0) {
-      LOG.error("\n\n ###### Stdout #######\n" + callback.getStdout().toString());
+      //LOG.error("\n\n ###### Stdout #######\n" + callback.getStdout().toString());
```

Comment:
Let's remove this condition, or maybe bring L243-L245 inside this conditional?

Reply:
These are debugging code for showing logs when there are IT failures.  I'll remove these changes.